assets/large_language_models/prompts/completions_natural_language_to_python/completions_generate_a_quiz/asset.yaml

```diff
@@ -0,0 +1,3 @@
+type: prompt
```

Comment:
The 'generate a quiz' prompt is in the wrong subdirectory (under the python prompt directory).  In the CELA review the quiz prompt was marked as "No for ignite".  Do you want to remove it?